pkg/autoupdate/autoupdate.go

```diff
@@ -316,7 +316,15 @@ func (t *task) localUpdateAvailable() (bool, error) {
 	if err != nil {
 		return false, err
 	}
-	return localImg.Digest().String() != t.image.Digest().String(), nil
+	// Go through the list of digests and compare to determine if there is a new image available
```

Comment:
`return localImg.ID != t.image.ID()` (maybe `.String()` is necessary) could be sufficient. An equal digest currently implies the same ID. (That will not be the case with https://github.com/containers/image/pull/1980 and zstd:chunked images, in principle … so this more complex check is potentially a bit more correct.)

Reply:
Fixed